arch/arm/configs/tsimx6ul_defconfig

```diff
@@ -1095,12 +873,12 @@ CONFIG_CRC_T10DIF=y
 CONFIG_CRC7=m
 CONFIG_DMA_CMA=y
 CONFIG_CMA_SIZE_MBYTES=64
-CONFIG_PRINTK_TIME=y
```

Comment:
Don't we want to leave time in on the full defconfigs?

Reply:
I'll add back a requirement for this.